src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -676,4 +676,45 @@ class JavaMigrations {
       .insert()
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "12.0.0-open", _))
   }
+
+  
+  @ChangeSet(order = "108", id = "108-add_corretto_11.0.3", author = "philiplourandos")
+  def migrate108(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "11.0.3-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.3.7.1/amazon-corretto-11.0.3.7.1-linux-x64.tar.gz", Linux64),
+      Version("java", "11.0.3-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.3.7.1/amazon-corretto-11.0.3.7.1-windows-x64.zip", Windows),
+      Version("java", "11.0.3-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.3.7.1/amazon-corretto-11.0.3.7.1-macosx-x64.tar.gz", MacOSX))
+      .validate()
+      .insert()
+    Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "11.0.2-amzn", _))
+  }
+
+  @ChangeSet(order = "109", id = "109-add_corretto_8.0.212", author = "philiplourandos")
+  def migrate109(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "8.0.212-amzn", "https://d3pxv6yz143wms.cloudfront.net/8.212.04.1/amazon-corretto-8.212.04.1-linux-x64.tar.gz", Linux64),
+      Version("java", "8.0.212-amzn", "https://d3pxv6yz143wms.cloudfront.net/8.212.04.1/amazon-corretto-8.212.04.1-windows-x64-jdk.zip", Windows),
+      Version("java", "8.0.212-amzn", "https://d3pxv6yz143wms.cloudfront.net/8.212.04.1/amazon-corretto-8.212.04.1-macosx-x64.tar.gz", MacOSX))
+      .validate()
+      .insert()
```

Comment:
Sorry if this wasn't clear in our earlier communication, but can you delete `8.0.202-amzn` after insertion of `8.0.212-amzn` please?